symfony/plugins/orangehrmAdminPlugin/Api/Model/SkillModel.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
skill model not related to employment status

Reply:
removed